src/patternfly/utilities/Text/text.scss

```diff
@@ -0,0 +1,77 @@
+// stylelint-disable
+
+// Font size options
+$pf-u-font-size-options: (
+  font-size-xs:  (font-size var(--pf-global--FontSize--xs)),
+  font-size-sm:  (font-size var(--pf-global--FontSize--sm)),
+  font-size-md:  (font-size var(--pf-global--FontSize--md)),
+  font-size-lg:  (font-size var(--pf-global--FontSize--lg)),
+  font-size-xl:  (font-size var(--pf-global--FontSize--xl)),
+  font-size-2xl: (font-size var(--pf-global--FontSize--2xl)),
+  font-size-3xl: (font-size var(--pf-global--FontSize--3xl)),
+  font-size-4xl: (font-size var(--pf-global--FontSize--4xl))
+);
+
+@include pf-utility-builder($pf-u-font-size-options, $pf-global--breakpoint-list);
+
+
+// Font weight options
+$pf-u-font-weight-options: (
+  font-weight-light:              (font-weight var(--pf-global--FontWeight--light)),
+  font-weight-normal:             (font-weight var(--pf-global--FontWeight--normal)),
+  font-weight-bold:               (font-weight var(--pf-global--FontWeight--bold)),
+  font-weight-semi-bold:          (font-weight var(--pf-global--FontWeight--semi-bold)),
```

Comment:
I don't think we need semi-bold or overpass-*, as those are only for overpass, which we don't need to support in the utility.